src/features/tauri/download-file.ts

```diff
@@ -0,0 +1,16 @@
+import { save } from '@tauri-apps/plugin-dialog'
+import { writeFile } from '@tauri-apps/plugin-fs'
+
+export const downloadFile = async (filename: string, blob: Blob) => {
```

Comment:
The provides a non tauri implementation as well. I'm wondering if we move this from `/tauri/` into an `/filesystem/` or `/download/` feature?